### PR TITLE
made Equation class

### PR DESCRIPTION
**Pull request by Arnav814**

### CODE REVIEW

File: Evaluator.java [hunk 1]
```diff
@@ -2,4 +2,16 @@ public class Evaluator {
...
+    public class Equation {
+        private int operand_1, operand_2;
+        private char operator;
+
+        public Equation(int operand_1, int operand_2, char operator) {
+            this.operand_1 = operand_1;
+            this.operand_2 = operand_2;
+            this.operator = operator;
+        }
+
```
Comment (sumitace) [inline Evaluator.java]: Please remove this extra space here. Otherwise it looks good

**Review by sumitace — CHANGES_REQUESTED** [review]
Minor space removal request

File: Evaluator.java (hunk 1 above)
Comment (Arnav814) [reply Evaluator.java]: okayyyyy fine

**Review by sumitace — APPROVED** [review]
lgtm 👍

**Pull request merged**